tools/benchmark_adaparseA_1_big_files.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Set the number of trials
+num_trials=50
+
+wget https://github.com/ada-url/url-various-datasets/raw/main/top100/top100.txt
+
+# File list to benchmark against
+files=("top100.txt") #"linux_files.txt" "wikipedia_100k.txt" 
```

Comment:
You could include all of them, it is fine...

Then you can download them as needed...

```bash
if [ -f "top100.txt" ]; then
    echo "top100.txt exists."
else 
    echo "downloading top100.txt."
    curl https://raw.githubusercontent.com/ada-url/url-various-datasets/main/top100/top100.txt -O
fi
```

Reply:
wget is also fine, but don't redownload when the file is already present.